<launch>
  <arg name="map_file" default="$(find bwi_guidance)/maps/map3.yaml" />
  <arg name="graph_file" default="$(find bwi_guidance)/maps/graph.yaml" />
  <node name="$(anon view_graph)" pkg="bwi_mapper" type="view_graph"
    args="$(arg map_file) $(arg graph_file)" />
</launch>
